cli/central-client/src/main/java/org/ballerinalang/central/client/CentralAPIClient.java

```diff
@@ -136,8 +142,9 @@ public Package getPackage(String orgNamePath, String packageNamePath, String ver
                     }
     
                     // If request sent is wrong or error occurred at remote repository
-                    if (getPackageResponse.code() == HttpsURLConnection.HTTP_BAD_REQUEST ||
-                        getPackageResponse.code() == HttpsURLConnection.HTTP_INTERNAL_ERROR) {
+                    if (getPackageResponse.code() == HTTP_BAD_REQUEST ||
+                        getPackageResponse.code() == HTTP_INTERNAL_ERROR ||
+                        getPackageResponse.code() == HTTP_BAD_GATEWAY) {
```

Comment:
I see that 502(HTTP_BAD_GATEWAY) and 503(HTTP_SERVICE_UNAVAILABLE) can be caused at the LB level as well, in which it returns an html payload without going to . That would cause an error when doing 
```java
Error error = new Gson().fromJson(body.get().string(), Error.class);
```

A sample response from LB:
```html
<html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>nginx</center>
</body>
</html>
```

Also 503 is not captured here.

Reply:
@hemikak Here we are only checking the 502, So if we add another if for 503, It will solve this issue right?
All 502 responses should have an error message, Is that the case?

Reply:
Added 503